include/ginkgo/core/base/math.hpp

```diff
@@ -908,7 +913,11 @@ GKO_INLINE GKO_ATTRIBUTES constexpr xstd::enable_if_t<is_complex_s<T>::value,
                                                       remove_complex<T>>
 abs(const T &x)
 {
+#ifdef CL_SYCL_LANGUAGE_VERSION
```

Comment:
Could you add some todo here? like `DPCPP currently needs to use OPENCL to support complex multiplication.`